### ISSUE TITLE
disparity validation set split

### ISSUE THREAD

**Issue by Candy-Crusher** [uzh-rpg/DSEC#65]
Hi dear authors, thx for the amazing work.
Could you plz enlight me how to split the disparity dataset for train and validation to replicate your baseline results?

**Comment by magehrig** [comment]
we did not train with a validation set. We just trained with the full training set and took the last checkpoint
